consumer/src/main/java/com/ihren/consumer/processor/PersonProcessor.java

```diff
@@ -0,0 +1,17 @@
+package com.ihren.consumer.processor;
+
+import com.ihren.consumer.sink.PersonSink;
+import com.ihren.model.Person;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+import java.util.function.Consumer;
+
+@Component
+@RequiredArgsConstructor
+public class PersonProcessor {
```

Comment:
You could implement Consumer<Person> in this implementation and don't call processPrson method in configuration